kolibri/plugins/learn/assets/src/views/LibraryPage/PinnedNetworkResources.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+
+  <KGrid>
+    <KGridItem
+      v-for="group in groups"
+      :key="group.id"
+    >
+      <KGridItem>
+        <h2>
+          <KIcon icon="device" />
+          <span class="device-name">{{ group.deviceName }}</span>
+        </h2>
+      </KGridItem>
+      <KGridItem
+        v-for="item in group.content"
+        :key="item.id"
+        :layout="{ span: cardColumnSpan, alignment: 'auto' }"
+      >
+        <CardContent
+          :content="item.title"
+          :body="item.body"
+        />
+      </KGridItem>
+    </KGridItem>
+  </KGrid>
+
+</template>
+
+
+<script>
+
+  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
```

Comment:
There is a useKResponsiveWindow composable that should be used in place of this mixin. Please see https://github.com/learningequality/kolibri/blob/1b5898310a58ce6e788d1d48a109b02ff29a41bb/kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue for a usage example